src/app/data-planes/components/data-plane-traffic/ServiceTrafficGroup.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <KCard
+    class="service-traffic-group"
+    :class="`type-${props.type}`"
+  >
+    <template #body>
+      <div class="body">
+        <slot name="default" />
+      </div>
+    </template>
+  </KCard>
+</template>
+<script lang="ts" setup>
+const props = defineProps<{
+  type: string
+}>()
+</script>
+<style lang="scss" scoped>
+.service-traffic-group .body {
+  display: flex;
+  flex-direction: column;
+  gap: $kui-space-40;
+}
+.type-passthrough,
+.type-outbound {
+  padding: $kui-space-40;
+  border-radius: $kui-border-radius-40;
+}
+.service-traffic-group {
+  &.type-passthrough {
+    /* FIXME(jc): These colors is in the design but don't seem to be in @kong/design-tokens */
+    border-color: #00D6A4;
+    background-color: #ECFFFB;
```

Comment:
@kleinfreund do you know if I'm missing something here, or do you have any suggestions? I can check in with design-tokens folks if not.

Reply:
If you're using colors not in the design system, they typically fall in two categories:

1. You're doing something custom; should you be? Maybe, maybe not.
2. The Figma design you're referencing hasn't been updating to utilize the tokens. Check with your designer

Reply:
Gotcha, let me check again but unless I'm misremembering the there was a green that was in the token set, but not with a semantic alias, so it looked like I couldn't get to it. let me look again and I'll come back to you.

(From what I remember the design also uses the non-semantic color name so `green-000` , 000 being whatever it was on the scale, again let me triple check)

Reply:
Here's the hexcode in design-tokens:

https://github.com/Kong/design-tokens/blob/c64b570fd649762bbbfb6937425852a18b663833/tokens/alias/color/index.json#L147-L149

and the Figma

![Screenshot 2023-11-24 at 15 10 24](https://github.com/kumahq/kuma-gui/assets/554604/0d897eee-d716-4171-8f5d-89e18ee95baa)

The border hex is the most important one. I guess we could just use a different color that exists in the semantic colors but it feels like since design are using these colors we should have access to them? I'm fine with that but we should also tell design not to use these.

---

I remember now I was unsure about the background one (which does exist) and I've added a commit to fill it out here. I think the not-terribly-important reason why I was unsure was: the color doesn't mean `success` it means 'green' i.e. we want to use 'green' for passthrough, not 'success' for passthrough. So if we were to expose the colors (see above) I would rather say 'green' over 'success' <- this one definitely isn't as important as not having the color at all.

Reply:
I think this is a case where it’s expected to have custom colors. We're not indicating something in a particular state/meaning the design system could know about here. We're arbitrarily assigning these greens to a card for passthrough traffic. Same thing for outbound traffic which happens to use colors present in the tokens.

Should these just generally use custom colors then?

Reply:
Hmm yeah quite possibly. We could even make a `--kuma-green-40` so at least we match up with the value/name in the design.

I think I would rather have access to `--kui-green-40` though, but I get why there's a reason to restrict people from using those (they should prefer semantic ones where possible). I think my main thing is, if design have access to it, and we need to implement the design, then we need access also.

> Same thing for outbound traffic which happens to use colors present in the tokens.

Also agree with this ^, although `primary` isn't as bad as using `success`

Lets see what @adamdehaven comes back with then

Reply:
Ok so that sounds like "please use the colors you have access to in the tokens"? i.e. we choose a different color from design-tokens that we do have access to?

Reply:
Just use the hex if you need to; Jason will be back next week and can ask him how to proceed if we need to expose some green tokens